openshift_pipelines/uninstalling-pipelines.adoc

```diff
@@ -0,0 +1,19 @@
+[id="uninstalling-pipelines"]
+= Uninstalling OpenShift Pipelines
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+:context: uninstalling-pipelines
+
+toc::[]
+
+Uninstalling the {pipelines-title} Operator is a two step process. First, you delete the custom resources, which are added by default when you install the Operator. Second, you uninstall the {pipelines-title} Operator.
+
+Simply uninstalling the Operator will not remove the {pipelines-title} components created by default when the Operator is installed.
+
+include::modules/op-deleting-the-pipelines-component-and-custom-resources.adoc[leveloffset=+1]
+
+include::modules/op-uninstalling-the-pipelines-operator.adoc[leveloffset=+1]
+
+== Additional Resources
+
+* You can learn more about uninstalling Operators on {product-title} in xref:../operators/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
```

Comment:
in the deleting Operators from a cluster section.?

Reply:
done